src/MSALWrapper.Test/TokenFetcherPublicClientTest.cs

```diff
@@ -371,6 +373,25 @@ public async Task GetTokenNormalFlowAsync_GetTokenInteractive_Throws_OperationCa
             tokenFetcher.ErrorsList[2].Message.Should().Be("Interactive Auth (with extra claims) timed out after 15 minutes.");
         }
 
+        /// <summary>
+        /// .

```

Comment:
We could probably use a better comment here than `.`.

Reply:
Good catch. I forgot to write the final comment😅